cmd/podman/common/create.go

```diff
@@ -588,6 +588,15 @@ func DefineCreateFlags(cmd *cobra.Command, cf *entities.ContainerCreateOptions,
 			"replace", false,
 			`If a container with the same name exists, replace it`,
 		)
+
+		decryptionKeysFlagName := "decryption-key"
+		createFlags.StringSliceVar(
+			&cf.DecryptionKeys,
+			decryptionKeysFlagName, []string{},
+			"*Experimental* key needed to decrypt the image (e.g. /path/to/key.pem)",
+		)
+		_ = cmd.RegisterFlagCompletionFunc(annotationFlagName, completion.AutocompleteNone)
+
```

Comment:
Remove this blank line to make linter happy.

Reply:
Fixed